.github/workflows/ci.yml

```diff
@@ -20,27 +26,66 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+      - name: Setup Java (temurin@8)
+        uses: actions/setup-java@v4
+        with:
+          distribution: temurin
+          java-version: 8
+          cache: sbt
       - uses: coursier/cache-action@v6
       - run: sbt docs/run
   plugin-tests:
     name: Compiler and SBT plugin tests
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [ ubuntu-latest ]
+        scala:
+          - 2.12.13
+          - 2.12.14
+          - 2.12.15
+          - 2.12.16
+          - 2.12.17
+          - 2.12.18
+          - 2.12.19
+          - 2.13.5
+          - 2.13.6
+          - 2.13.7
+          - 2.13.8
+          - 2.13.9
+          - 2.13.10
+          - 2.13.11
+          - 2.13.12
+          - 2.13.13
+          - 2.13.14
```

Comment:
Ideally, these should come from the build definition. But adding support for any new Scala 2 versions is done barehand not just adding 1 line into the `build.sbt`. So, let's make it gradually